armadillo/src/main/java/at/favre/lib/armadillo/DefaultEncryptionProtocol.java

```diff
@@ -159,6 +160,17 @@ public String deriveContentKey(String originalContentKey) {
         }
     }
 
+    @Override
+    public void setKeyStretchingFunction(KeyStretchingFunction function) {
+        Objects.requireNonNull(function);
+        keyStretchingFunction = function;
```

Comment:
Are there any issues by being able to change the `keyStretchingFunction` after creation of the protocol?

Reply:
I don't see any issue..